controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -161,6 +162,15 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 			return reconcile.Result{}, err
 		}
 
+		clientOperatorVersion, err := semver.Parse(r.storageConsumer.Status.Client.OperatorVersion)
+		if err != nil {
+			return reconcile.Result{}, fmt.Errorf("malformed ClientOperatorVersion: %v", err)
+		}
```

Comment:
Do we have a guarantee that his exists from the second the consumer is enabled? Or can it be that this value will be empty until the first heartbeat?

Reply:
empty until first hearbeat, in earlier revision I was guarding this w/ `allowRemoteStorageConsumers` as older clients would've this set and version is always available.

Reply:
added check for emptiness.